ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -59,14 +59,20 @@
   }
 }
 
+.columns-tab {
```

Comment:
Please move these css to styles.css as it should be same across the components use in the UI

Reply:
updated